### PR TITLE
Adjust Notification Padding Tokens

### PR DESCRIPTION
**Pull request by jeaniehuynh**
### Platforms Impacted
- [x] iOS
- [ ] macOS

### Description of changes

The bounds of a couple elements were off in the figma, so the tokens were slightly wrong. This change adjusts them to what they should be!

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Simulator Screen Shot - iPad Pro (12 9-inch) (5th generation) - 2022-08-12 at 10 55 20 | Simulator Screen Shot - iPad Pro (12 9-inch) (5th generation) - 2022-08-12 at 13 31 13 |
| Simulator Screen Shot - iPad Pro (12 9-inch) (5th generation) - 2022-08-12 at 10 55 24 | Simulator Screen Shot - iPad Pro (12 9-inch) (5th generation) - 2022-08-12 at 13 31 20 |

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] iPad [Pointer interaction](https://developer.apple.com/documentation/uikit/pointer_interactions)
- [ ] [SwiftUI](https://developer.apple.com/tutorials/swiftui) consumption (validation or new demo scenarios needed)
- [ ] Objective-C exposure (provide it only if needed)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/fluentui-apple/pull/1159)

### CODE REVIEW

File: ios/FluentUI/Notification/NotificationTokens.swift [hunk 1]
```diff
@@ -170,13 +170,13 @@ open class NotificationTokens: ControlTokens {
     /// The value for the bottom padding between the notification and its anchor view
     open var bottomPresentationPadding: CGFloat { 20.0 }
     /// The value for the horizontal padding between the elements within a notification and its frame
-    open var horizontalPadding: CGFloat { 19.0 }
+    open var horizontalPadding: CGFloat { 16.0 }
```
Comment (huwilkes) [inline ios/FluentUI/Notification/NotificationTokens.swift:173]: If these line up with our spacing tokens, could we switch to those?

**Review by huwilkes — APPROVED** [review]

File: ios/FluentUI/Notification/NotificationTokens.swift [hunk 2]
```diff
@@ -170,13 +170,13 @@ open class NotificationTokens: ControlTokens {
     /// The value for the bottom padding between the notification and its anchor view
     open var bottomPresentationPadding: CGFloat { 20.0 }
```
Comment (harrieshin) [inline ios/FluentUI/Notification/NotificationTokens.swift:171]: this is a large too?

File: ios/FluentUI/Notification/NotificationTokens.swift [hunk 3]
```diff
@@ -168,15 +168,15 @@ open class NotificationTokens: ControlTokens {
     }
 
     /// The value for the bottom padding between the notification and its anchor view
-    open var bottomPresentationPadding: CGFloat { 20.0 }
+    open var bottomPresentationPadding: CGFloat { globalTokens.spacing[.medium] }
```
Comment (harrieshin) [inline ios/FluentUI/Notification/NotificationTokens.swift:171]: is it meant to be 16? or is it meant to be 20?

Comment (jeaniehuynh) [reply ios/FluentUI/Notification/NotificationTokens.swift:171]: 16!

image

File: ios/FluentUI/Notification/NotificationTokens.swift [hunk 4]
```diff
@@ -168,15 +168,15 @@ open class NotificationTokens: ControlTokens {
...
+    open var verticalPadding: CGFloat { globalTokens.spacing[.small] }
     /// The value for the horizontal padding between the elements within a single-line notification and its frame
-    open var verticalPaddingForOneLine: CGFloat { 18.0 }
+    open var verticalPaddingForOneLine: CGFloat { globalTokens.spacing[.medium] }
     /// The value for the horizontal spacing between the elements within a notification
-    open var horizontalSpacing: CGFloat { 19.0 }
+    open var horizontalSpacing: CGFloat { globalTokens.spacing[.medium] }
     /// The value for the minimum height of a multi-line notification
     open var minimumHeight: CGFloat { 64.0 }
     /// The value for the minimum height of a single-line notification
```
Comment (joannaquu) [inline ios/FluentUI/Notification/NotificationTokens.swift:182]: minimumHeightForOneLine is never used

Comment (joannaquu) [inline ios/FluentUI/Notification/NotificationTokens.swift:182]: minimumHeightForOneLine is never used

**Comment by harrieshin** [comment]
minimumHeightForOneLine still correctly increase larger text and longer message that will be more that one line?  would you midn updating your before / afterscreenshots please?

**Review by sophialee0416 — APPROVED** [review]

**Comment by joannaquu** [comment]
> > minimumHeightForOneLine still correctly increase larger text and longer message that will be more that one line? would you midn updating your before / afterscreenshots please?
> 
> @harrieshin Screenshots updated! Let me actually confirm the minimum height specs with design team. The figma doesn't specify heights for one-line scenario.

Ian just updated the figma so it should be 62 for multiline and 52 for single line. https://www.figma.com/file/ArWQiU7BAlrDGNmbrGEXbA/Fluent-2-iOS-specs?node-id=3165%3A48344

**Comment by jeaniehuynh** [comment]
> > > minimumHeightForOneLine still correctly increase larger text and longer message that will be more that one line? would you midn updating your before / afterscreenshots please?
> > 
> > 
> > @harrieshin Screenshots updated! Let me actually confirm the minimum height specs with design team. The figma doesn't specify heights for one-line scenario.
> 
> Ian just updated the figma so it should be 62 for multiline and 52 for single line.

62 was just the height for that specific example in the figma, but the _minimum_ should be 52 for all toasts!
image

**Pull request merged**